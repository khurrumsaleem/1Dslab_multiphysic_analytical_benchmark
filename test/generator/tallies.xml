<?xml version='1.0' encoding='utf-8'?>
<tallies>
  <filter id="1" type="material">
    <bins>1</bins>
  </filter>
  <filter id="2" type="energy">
    <bins>0.0 20000000.0</bins>
  </filter>
  <filter id="11" type="energyout">
    <bins>0.0 20000000.0</bins>
  </filter>
  <filter id="12" type="legendre">
    <order>3</order>
  </filter>
  <tally id="15">
    <filters>1 2</filters>
    <nuclides>total</nuclides>
    <scores>flux total absorption nu-fission fission</scores>
    <estimator>tracklength</estimator>
  </tally>
  <tally id="23">
    <filters>1 2</filters>
    <nuclides>total</nuclides>
    <scores>flux nu-fission</scores>
    <estimator>analog</estimator>
  </tally>
  <tally id="17">
    <filters>1 2 11 12</filters>
    <nuclides>total</nuclides>
    <scores>nu-scatter</scores>
    <estimator>analog</estimator>
  </tally>
  <tally id="20">
    <filters>1 2 11</filters>
    <nuclides>total</nuclides>
    <scores>nu-scatter scatter</scores>
    <estimator>analog</estimator>
  </tally>
  <tally id="22">
    <filters>1 11</filters>
    <nuclides>total</nuclides>
    <scores>nu-fission</scores>
    <estimator>analog</estimator>
  </tally>
</tallies>
