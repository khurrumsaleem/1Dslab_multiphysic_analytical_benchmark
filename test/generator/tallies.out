 ===========================>     TALLY 15     <============================

 Material 1
   Incoming Energy [0, 20000000)
     Total Material
       Flux                                 0.0747583 +/- 0.000421537
       Total Reaction Rate                  13.3605 +/- 0.0838156
       Absorption Rate                      1.00903 +/- 0.00509497
       Nu-Fission Rate                      1.88874 +/- 0.00865243
       Fission Rate                         0.739457 +/- 0.00338686
 ===========================>     TALLY 23     <============================

 Material 1
   Incoming Energy [0, 20000000)
     Total Material
       Flux                                 0.0747942 +/- 0.000329503
       Nu-Fission Rate                      1.88747 +/- 0.0148323
 ===========================>     TALLY 17     <============================

 Material 1
   Incoming Energy [0, 20000000)
     Outgoing Energy [0, 20000000)
       Legendre expansion, P0
         Total Material
           Scattering Production Rate           12.3723 +/- 0.0695863
       Legendre expansion, P1
         Total Material
           Scattering Production Rate           2.70448 +/- 0.0169985
       Legendre expansion, P2
         Total Material
           Scattering Production Rate           1.10927 +/- 0.00672876
       Legendre expansion, P3
         Total Material
           Scattering Production Rate           0.565578 +/- 0.00653358
 ===========================>     TALLY 20     <============================

 Material 1
   Incoming Energy [0, 20000000)
     Outgoing Energy [0, 20000000)
       Total Material
         Scattering Production Rate           12.3723 +/- 0.0695863
         Scattering Rate                      12.3689 +/- 0.0694799
 ===========================>     TALLY 22     <============================

 Material 1
   Outgoing Energy [0, 20000000)
     Total Material
       Nu-Fission Rate                      1.88747 +/- 0.0148323
